typescript/src/schema/proto.ts

```diff
@@ -393,22 +392,26 @@ function getMethodConfig(
 ): protos.grpc.service_config.MethodConfig {
   let exactMatch: protos.grpc.service_config.IMethodConfig | undefined;
   let serviceMatch: protos.grpc.service_config.IMethodConfig | undefined;
-  for (const config of grpcServiceConfig.methodConfig) {
-    if (!config.name) {
-      continue;
-    }
-    for (const name of config.name) {
-      if (name.service === serviceName && !name.method) {
-        serviceMatch = config;
+  if (Array.isArray(grpcServiceConfig.methodConfig)) {
+    for (const config of grpcServiceConfig.methodConfig) {
+      if (!config.name) {
+        continue;
       }
-      if (name.service === serviceName && name.method === methodName) {
-        exactMatch = config;
+      for (const name of config.name) {
+        if (name.service === serviceName && !name.method) {
```

Comment:
Seems kind of strange to me that this wouldn't be an if/else, because would it be possible for !name.method and name.method ==== methodName?

Reply:
Agreed, if-else is more logical here. Fixed.